internal/grpc/interceptors/recovery/recovery.go

```diff
@@ -45,8 +45,8 @@ func NewStream() grpc.StreamServerInterceptor {
 }
 
 func recoveryFunc(ctx context.Context, p interface{}) (err error) {
-	stack := debug.Stack()
+	debug.PrintStack()
```

Comment:
@glpatcern can you revert this change? Even though is useful it will break log parsing. Let's create a github issue to proper address it.

Reply:
Interestingly, Ishank marked this "nice", and was motivated by my comment above: "I got a full stack printed with literal \n's all over!". Why would this break log parsing? Because it's multiline?

Reply:
Opened #1054 to track this.